lib/y-richtext.coffee

```diff
@@ -100,7 +101,9 @@ class YRichText extends BaseClass
   #   update the position of our cursor to the new one using a character
   #   @param character [Character] the new character
   updateCursorPosition : (obj) => @locker.try ()=>
-    if typeof obj is "number"
+    if obj is misc.LAST_CHAR
+      @selfCursor = misc.LAST_CHAR
+    else if typeof obj is "number"
       @selfCursor = @_model.getContent("characters").ref(obj)
```

Comment:
`@_model.getContent("characters").ref(obj)` returns `null` if this is the last position